build_and_test.sh

```
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+echo "🚀 Building SnapClone iOS App with Google Sign-In + Snapchat Integration"
+echo "=================================================================="
+
+# Navigate to the iOS project
+cd ios/SnapCloneApp
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)